File: MPI/mpicheck/tests/functions/3.0/MPI_Neighbor_alltoallv.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const void *var_0;
    const int var_1[2];
    const int var_2[2];
    MPI_Datatype var_3;
    void *var_4;
    const int var_5[2];
    const int var_6[2];
    MPI_Datatype var_7;
    MPI_Comm var_8;
    int ret;
    /* calls */
    ret = MPI_Neighbor_alltoallv(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8);
    ret = PMPI_Neighbor_alltoallv(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8);
    return 0;
}

File: MPI/mpicheck/tests/functions/3.0/MPI_Neighbor_alltoallv.f

        
        program main
        include 'mpif.h'
        
        TYPE(INTEGER) var_0(10)
       INTEGER var_1(10)
       INTEGER var_2(10)
       INTEGER var_3
       TYPE(INTEGER) var_4(10)
       INTEGER var_5(10)
       INTEGER var_6(10)
       INTEGER var_7
       INTEGER var_8
       INTEGER var_9
        call mpi_neighbor_alltoallv(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, &
       var_9)
       call pmpi_neighbor_alltoallv(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, &
       var_8, var_9)
        end program main
    